include/ginkgo/core/base/abstract_factory.hpp

```diff
@@ -237,6 +237,22 @@ class enable_parameters_type {
 public:
     using factory = Factory;
 
+    /**
+     * Loggers to be attached to the factory and generated object.
+     */
+    std::vector<std::shared_ptr<const log::Logger>> loggers{};
```

Comment:
private members?

Reply:
I would agree that it's not necessary to put it as a public member.